src/modules/SX127x/SX127x.cpp

```diff
@@ -96,12 +96,17 @@ int16_t SX127x::beginFSK(uint8_t chipVersion, float br, float freqDev, float rxB
   state = SX127x::setAFCBandwidth(rxBw);
   RADIOLIB_ASSERT(state);
 
-  // set AFC&AGC trigger to RSSI (both in OOK and FSK)
-  state = SX127x::setAFCAGCTrigger(RADIOLIB_SX127X_RX_TRIGGER_RSSI_INTERRUPT);
+  // set AFC&AGC trigger to RSSI (in OOK) or both (in FSK)
+  if(enableOOK) {
+    state = SX127x::setAFCAGCTrigger(RADIOLIB_SX127X_RX_TRIGGER_RSSI_INTERRUPT);
+  } else {
+    state = SX127x::setAFCAGCTrigger(RADIOLIB_SX127X_RX_TRIGGER_BOTH);
+  }
+
   RADIOLIB_ASSERT(state);
 
   // enable AFC
-  state = SX127x::setAFC(false);
+  state = SX127x::setAFC(true);
   RADIOLIB_ASSERT(state);
```

Comment:
The disabled AFC and AFC/AGC triggers are a workaround for https://github.com/jgromes/RadioLib/issues/505, the PR would overwrite those. I think you might be merging from some older version.